stash_datacite/lib/stash/import/dryad_manuscript.rb

```diff
@@ -0,0 +1,68 @@
+module Stash
+  module Import
+    class DryadManuscript
+
+      def initialize(resource:, httparty_response:)
+        @resource = resource
+        @response = httparty_response.with_indifferent_access # so you can use symbols instead of strings for access
+      end
+
+      def populate
+        populate_title
+        populate_authors
+        populate_abstract
+        populate_keywords
+      end
+
+      def populate_title
+        return if @response[:title].blank?
+        @resource.update(title: @response[:title])
+      end
+
+      def populate_authors
+        return if @response[:authors].blank? || @response[:authors][:author].blank?
+        authors = @response[:authors][:author]
+        authors.each do |api_author|
+          author = @resource.authors.create(
+            author_first_name: api_author['givenNames'],
+            author_last_name: api_author['familyName'],
+            author_orcid: (api_author['identifierType'] == 'orcid' ? api_author[:identifier] : nil)
+          )
+          update_email(db_author: author)
```

Comment:
Do you get affiliation info from this API? If so I can tie in the ROR stuff afterward

Reply:
It doesn't seem like there is any affiliation information in the Dryad manuscript API.  There is often some publisher information for the paper, but usually the researchers are not affiliated with the paper publisher.

I'm not sure where Ryan is getting his information, maybe from the publishers?  He might be able to prod some to put it in somewhere if it was an option for them.